sys/drv/bcm2835_gpio.c

```diff
@@ -21,9 +26,14 @@ static void delay(int64_t count) {
 }
 
 #define NGPIO_PIN 54
+#define FDT_PINS_INVAL (-1)
 
 /* clang-format on */
 
+typedef struct bcm2835_gpio {
+  resource_t *gpio;
+} bcm2835_gpio_t;
+
 void bcm2835_gpio_function_select(resource_t *r, unsigned pin,
```

Comment:
This function is no longer global. Please remove the signature from the header and add `static`.

Reply:
Done.